### PR TITLE
Remove semantic coupling in metadata spec

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php [hunk 1]
```diff
@@ -32,10 +29,10 @@ function let()
                 'driver' => 'doctrine/orm',
                 'templates' => 'SyliusProductBundle:Product',
                 'classes' => [
-                    'model' => Product::class,
+                    'model' => 'Sylius\Component\Product\Model\Product',
```
Comment (pamil) [inline src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php]: IMHO it is still coupled, we should avoid any reference to existing components. Can we use `\Example\Resource` instead here and something similar everything else?

Comment (lchrusciel) [inline src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php]: IMHO real example is much more readable. Should it be changed? ping @michalmarcinkowski @pjedrzejewski

Comment (pjedrzejewski) [inline src/Sylius/Component/Resource/spec/Metadata/MetadataSpec.php]: Yup.

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**